lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -21,15 +19,14 @@ import ErrorModal from './ErrorModal';
  * @param {OAuth2RedirectErrorAppProps} props
  */
 export default function OAuth2RedirectErrorApp({ location = window.location }) {
-  const { OAuth2RedirectError = /** @type {OAuthErrorConfig} */ ({}) } =
-    useContext(Config);
-
   const {
-    authUrl = null,
-    errorCode,
-    errorDetails = '',
-    canvasScopes = /** @type {string[]} */ ([]),
-  } = OAuth2RedirectError;
+    OAuth2RedirectError: {
+      authUrl = null,
+      errorCode,
+      errorDetails = '',
+      canvasScopes = /** @type {string[]} */ ([]),
+    },
+  } = useConfig(['OAuth2RedirectError']);
```

Comment:
The `OAuth2RedirectError` key should always be set by the backend when the corresponding app mode is set. See `enable_oauth2_redirect_error_mode` in the backend.